src/rbush-find.ts

```diff
@@ -1,44 +1,42 @@
-import { Polygon, Properties, Feature, MultiPolygon } from '@turf/helpers'
-import { RBush } from 'geojson-rbush'
-import intersect from '@turf/intersect'
-import area from '@turf/area'
-import { IArea } from './types'
-
-export const rbushFind = <I, O extends IArea>(
-  geojsonMultiPolygon: Feature<Polygon | MultiPolygon>,
-  tree: RBush<Polygon | MultiPolygon, Properties>,
-  mapper: (input: I) => O
-): { properties: O }[] => {
-  if (!geojsonMultiPolygon.properties) {
-    geojsonMultiPolygon.properties = {}
-  }
-
+import turfIntersect from '@turf/intersect'
+import turfArea from '@turf/area'
+import { IArea, IGeojson, IRBushTree } from './types'
+
+const rbushFind = <I, O extends IArea>(
+  geojson: IGeojson,
+  tree: IRBushTree,
+  format?: (input: I) => O
+) => {
   try {
+    if (!geojson.properties) {
+      geojson.properties = {}
+    }
     // Recherche le périmètre dans l'index bbox
-    const { features: matchingCommunes = [] } = tree.search(geojsonMultiPolygon)
+    const { features } = tree.search(geojson)
 
     // Filtre les éléments trouvés par l'index et compare le périmètre
-    return matchingCommunes.reduce((communes: { properties: O }[], commune) => {
-      const intersected = intersect(geojsonMultiPolygon, commune)
-      if (!intersected) return communes
+    return features.reduce((areas: O[], feature) => {
+      if (!feature.properties) return areas
 
-      const { properties } = commune
+      const intersection = turfIntersect(geojson, feature)
```

Comment:
J’avais volontairement laissé les `properties` car c’est un sous ensemble du format `GeoJson`. 
Adrien m’a parlé d’une éventuelle future fonctionnalité "afficher les forêts sur la carte d’un titre", et donc si demain `geo-api` doit retourner les périmètres c’était déjà prêt.

Reply:
@francoisromain